src/pages/snippets/v15-new-image-directive.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: New Image Directive in Angular v15
+description: "Example of New Image Directive in Angular 15 "
+tags: ["angular15", "image directive"] # comma separated list which gets trimmed
+pubDate: Mar 5, 2023
+contributedBy: "@deepakrudrapaul"
+---
+
+Here is a quick example of how to use the _NgOptimizedImage_ directive.
```

Comment:
I would suggest saying this is a `simple` example as there is a LOT of configuration possible that will be shown as console warnings when the app starts up

https://angular.io/guide/image-directive#performance-features
https://angular.io/guide/image-directive#configuring-an-image-loader-for-ngoptimizedimage

Reply:
Good idea changed it _simple_. Thanks

Reply:
I can still see this as quick.

Reply:
I have pushed the changes. Thanks